### PR TITLE
Welcome page, very barebones and minimal, but can be built upon

### PR DESCRIPTION
**Pull request by iamnukee**

### CODE REVIEW

**Comment by iamnukee** [comment]
[https://media.giphy.com/media/L3olFMMOvQVTCdYmI9/giphy.gif](url)

File: src/app/components/home/home.component.html [hunk 1]
```diff
@@ -1 +1,22 @@
-<h1>Welcome to EDCS</h1>
+<section class="intro">  
+    <div class="content">
+        <h1>{{page.subtitle}}</h1>
```
Comment (mll8657) [inline src/app/components/home/home.component.html]: Maybe manually type this in for consistency

File: src/app/components/home/home.component.scss [hunk 2]
```diff
@@ -0,0 +1,134 @@
+/* @import '~@angular/material/prebuilt-themes/indigo-pink.css';
```
Comment (mll8657) [inline src/app/components/home/home.component.scss]: clean comments?

File: src/app/components/header/header.component.html [hunk 3]
```diff
@@ -1,5 +1,5 @@
 
-<mat-toolbar class="mat-elevation-z2 edcs-header" color="primary">
+<mat-toolbar mat-elevation-z2 edcs-header color="primary">
```
Comment (Jss7268) [inline src/app/components/header/header.component.html]: What is this change for?

Comment (iamnukee) [reply src/app/components/header/header.component.html]: lol, nothing, it was a mistake I didn't catch

**Comment by Jss7268** [comment]
I'd suggest improving accessibility in terms of contrast better. Orange doesn't look great as a font either, maybe it could be white font inside of an orange box, or a sky blueish background with white font color.
Also, the font should be sans serif (Nothing on the site should be serif)

**Comment by iamnukee** [comment]
I felt orange was way too much for the background, but I will try a different color scheme and send a picture/video

**Comment by iamnukee** [comment]
Welcome Page2

File: src/app/components/home/home.component.scss [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    &:hover{ 
+      opacity: 0.8;
+    }
+    
+    &:active {
+      top: 1px;
+    }
+  }
+
+  footer {
```
Comment (iamnukee) [inline src/app/components/home/home.component.scss]: Currently not in use but may be beneficial for a more polished look to our website (not very important though)

**Comment by iamnukee** [comment]
Welcome Page2
Welcome Page2

**Pull request merged**